bundles/umljava/tools.vitruv.applications.umljava.java2uml/src/tools/vitruv/applications/umljava/java2uml/JavaToUmlClassifier.reactions

```diff
@@ -272,7 +273,10 @@ routine addUmlClassImplement(java::Class jClass, java::TypeReference jReference,
     }
     action {
         execute {
-            if (uInterface !== null) {
+            val matchingInterfaceRealization = uClass.interfaceRealizations.filter[name == uInterface.name + uInterface.name].filter[contract == uInterface].claimOneOrNone
```

Comment:
Is `uInterface.name + uInterface.name` correct?

Reply:
The Java to PCM reactions used this naming scheme for interface realizations. I changed it to just the name of the interface and removed any interface realization matching based on the realization name in the latest commit.